neo/Network/P2P/Payloads/GetFullBlocksPayload.cs

```diff
@@ -0,0 +1,37 @@
+using Neo.IO;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class GetFullBlocksPayload : ISerializable
+    {
+        private const ushort MaxBlocksCount = 500;
+        public uint IndexStart;
+        public ushort Count;
+
+        public int Size => sizeof(uint) + sizeof(ushort);
+
+        public static GetFullBlocksPayload Create(uint index_start, ushort count = 500)
```

Comment:
I prefer to remove the default value `500`. Blocks are very large, we should clearly indicate how many blocks we need, and we should not let other nodes send up to 500 blocks each time.